index.html

```diff
@@ -80,11 +82,27 @@ <h1>Visualization explanation</h1>
 
       <h1>Data Analysis</h1>
       <p><em>Summary of data, data types, and data preprocessing.</em></p>
+      <p>According to the data, the time it takes to cross from one side of the park to the other
+        without jaywalking is almost 11 times greater than crossing the street directly. (based on avg_travel_times.csv)
+        However, the average time it takes to cross a crosswalk is quite similar to that of jaywalking, but is
+        much safer. Even if people cross when there is no signal to walk, it is still safer than no crosswalk.
```

Comment:
We will need to make this more detailed (what data we use for each part of the visualization and how we used it) but that might be something we can do as we go